src/main/java/uk/gov/companieshouse/disqualifiedofficers/delta/consumer/DisqualifiedOfficersDeltaConsumer.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.companieshouse.disqualifiedofficers.delta.consumer;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.messaging.Message;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.delta.ChsDelta;
+import uk.gov.companieshouse.disqualifiedofficers.delta.processor.DisqualifiedOfficersDeltaProcessor;
+import uk.gov.companieshouse.logging.Logger;
+
+
+@Component
+public class DisqualifiedOfficersDeltaConsumer {
```

Comment:
Also unit tests for this class?

Reply:
The main topic is loosely tested in the integration tests in DisqualifiedOfficersDeltaConsumerITest.java. No test currently for the retry topic but it should be possible to set one up in here. Again there's no unit test in insolvency for this class.